### PR TITLE
Linting with flake8

### PR DESCRIPTION
**Pull request by inverse**
Leading on from the CI improvements adding flake8 + isort to lint files.

Some overlap with #11 but will enforce isort on CI now.

### CODE REVIEW

File: tell_me_your_secrets/__main__.py [hunk 1]
```diff
@@ -42,30 +47,17 @@
 argument_parser.add_argument('-g', '--gitignore', help='Ignore .gitignore mapped objects. ', action='store_true')
 module_logger = create_logger(MODULE_NAME)
 
-# Options
```
Comment (inverse) [inline tell_me_your_secrets/__main__.py:45]: I can leave this back but I Felt like it was redundant with the argument config

**Comment by inverse** [comment]
@valayDave any feedback for this one?

File: tell_me_your_secrets/__main__.py (hunk 1 above)
Comment (valayDave) [reply tell_me_your_secrets/__main__.py:45]: We can remove options.  I wanted to keep the process comments to ensure faster readability of code.

Comment (inverse) [reply tell_me_your_secrets/__main__.py:45]: Sure will update

**Pull request merged**